packages/cli/src/cmds/update.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Result, FxError, err, ok } from "@microsoft/teamsfx-api";
+import path from "path";
+import { Argv } from "yargs";
+import activate from "../activate";
+import CliTelemetry, { makeEnvRelatedProperty } from "../telemetry/cliTelemetry";
+import {
+  TelemetryEvent,
+  TelemetryProperty,
+  TelemetrySuccess,
+} from "../telemetry/cliTelemetryEvents";
+import { getSystemInputs } from "../utils";
+import { YargsCommand } from "../yargsCommand";
+
+export class UpdateAadApp extends YargsCommand {
+  public readonly commandHead = "aad-app";
+  public readonly command = this.commandHead;
+  public readonly description = "Update the Aad App Manifest in the current application.";
+
+  public builder(yargs: Argv): Argv<any> {
+    this.params = {
+      folder: {
+        describe: "Select root folder of the project",
+        string: true,
+      },
+      env: {
+        describe: "Select an existing environment for the AAD manifest",
+        requiresArg: true,
+        string: true,
+      },
+    };
+    return yargs.options(this.params).version(false);
+  }
+
+  public async runCommand(args: { [argName: string]: string }): Promise<Result<null, FxError>> {
+    const rootFolder = path.resolve((args.folder as string) || "./");
+    CliTelemetry.withRootFolder(rootFolder).sendTelemetryEvent(
+      TelemetryEvent.UpdateAadManifestStart
+    );
+    const resultFolder = await activate(rootFolder);
+    if (resultFolder.isErr()) {
+      CliTelemetry.sendTelemetryErrorEvent(TelemetryEvent.UpdateAadManifest, resultFolder.error);
+      return err(resultFolder.error);
+    }
+    const core = resultFolder.value;
+    const inputs = getSystemInputs(rootFolder, args.env);
+    inputs.ignoreEnvInfo = false;
+
+    // Update the aad manifest
+    const result = await core.deployAadManifest(inputs);
+    if (result.isErr()) {
+      CliTelemetry.sendTelemetryErrorEvent(
+        TelemetryEvent.UpdateAadManifest,
+        result.error,
+        makeEnvRelatedProperty(rootFolder, inputs)
+      );
+
+      return err(result.error);
+    }
+    CliTelemetry.sendTelemetryEvent(TelemetryEvent.UpdateAadManifest, {
+      [TelemetryProperty.Success]: TelemetrySuccess.Yes,
+      ...makeEnvRelatedProperty(rootFolder, inputs),
+    });
+    return ok(null);
+  }
+}
+
+export default class Update extends YargsCommand {
+  public readonly commandHead = "update";
+  public readonly command = `${this.commandHead} <application-manifest>`;
+  public readonly description = "Update the specific application manifest file.";
```

Comment:
@jidddddd please help review this Descrption also, thanks

Reply:
Is this for Teams app? If so, should be "Update the Teams application manifest file."

Reply:
Not really, update is the main command, both "teams app" and "aad-app" are sub-command.
For example, the command add feature in the v2 version, add is the main command, and add sso is the sub-command.